package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "perf-goggles",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:csd713/perf-goggles.git",
+  "author": "Chiranjeevi <jeevi146@gmail.com>",
```

Comment:
Do you mind keeping my original authorship? As I will have to maintain it long-term.